strings.po

```diff
@@ -13864,17 +13896,17 @@ msgstr "Ein medizinischer Entstinker."
 #. STRINGS.ITEMS.PILLS.VITAMINSUPPLEMENT.DESC
 msgctxt "STRINGS.ITEMS.PILLS.VITAMINSUPPLEMENT.DESC"
 msgid "A health <style=\"medicine\">Supplement</style> for improving <style=\"disease\">Immune System</style> strength."
-msgstr ""
+msgstr "Eine Gesundheits<style=\"medicine\">ergänzung</style> zur Stärkung des <style=\"disease\">Immunsystems</style>."
```

Comment:
<style> bitte immer um ganze Wörter ziehen

Reply:
An dieser Stelle dann "Gesundheitsergänzung" in <style>, oder eher den Satz umbauen?

Reply:
Ja, einfach das ganze Wort rein